test_programs/noir_test_success/mock_oracle/src/main.nr

```diff
@@ -151,3 +151,20 @@ fn test_mock_struct_field() {
         assert_eq(20, struct_field(point, another_array));
     }
 }
+
+#[test]
+fn test_times_called() {
+    // Safety: testing context
+    unsafe {
+        let mock = OracleMock::mock("void_field").returns(10);
+        assert_eq(mock.times_called(), 0);
+
+        assert_eq(void_field(), 10);
+        assert_eq(void_field(), 10);
+        assert_eq(void_field(), 10);
+        assert_eq(void_field(), 10);
+        assert_eq(void_field(), 10);
+
+        assert_eq(mock.times_called(), 5);
```

Comment:
Possibly dumb question - what would the result of this be if it were called from a brillig function? Would the count be reset there or carry over? And would the final check here include the times it was called in brillig?